src/serviceWorker.es6

```diff
@@ -0,0 +1,30 @@
+const CACHE_NAME = 'v1'
```

Comment:
Tohle vypadá tak, že při updatu sw se navýší tohle číslo tj. se zahodí cache. A to IMO nechceme...